src/Altinn.App.Core/Internal/Process/EventHandlers/ProcessTask/AbandonTaskEventHandler.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Altinn.App.Core.Features;
+using Altinn.App.Core.Internal.Process.ProcessTasks;
+using Altinn.Platform.Storage.Interface.Models;
+
+namespace Altinn.App.Core.Internal.Process.EventHandlers.ProcessTask
+{
+    /// <summary>
+    /// This event handler is responsible for handling the abandon event for a process task.
+    /// </summary>
+    public class AbandonTaskEventHandler : IAbandonTaskEventHandler
+    {
+        private readonly IEnumerable<IProcessTaskAbandon> _processTaskAbondons;
+
+        /// <summary>
+        /// This event handler is responsible for handling the abandon event for a process task.
+        /// </summary>
+        /// <param name="processTaskAbondons"></param>
+        public AbandonTaskEventHandler(IEnumerable<IProcessTaskAbandon> processTaskAbondons)
```

Comment:
This code used to be in DefaultTasksEvents. Common "abandon task" logic for all process tasks.